Source/events/EventConverters.ts

```diff
@@ -46,6 +46,21 @@ export class EventConverters {
         return uncommittedEvent;
     }
 
+    /**
+     * Creates an uncommitted embedding event (aka an event without an eventsourceid) from given parameters.
+     * @param {*} event Event content to constructor with.
+     * @param {EventType} eventType The event type.
+     * @param {boolean} isPublic Whether or not it is a public event
+     * @returns {UncommittedEvent} Constructed uncommitted event.
+     */
+    static getUncommittedEmbeddingEventFrom(event: any, eventType: EventType, isPublic: boolean): PbUncommittedEvent {
+        const uncommittedEvent = new PbUncommittedEvent();
+        uncommittedEvent.setArtifact(eventTypes.toProtobuf(eventType));
+        uncommittedEvent.setPublic(isPublic);
+        uncommittedEvent.setContent(JSON.stringify(event));
+        return uncommittedEvent;
+    }
```

Comment:
I don't really see why we need this. If so it does not really belong here in this class

Reply:
Events committed in the `@compare` or `@deleteMethod` methods don't have an EventSourceId tied to them, as those methods just return objects. Usually when committing events, you'd do `eventStore.commit(new DishPrepared('blah'), 'eventsourceid')`, but you don't know what the eventsourceid is in the compare method and you don't need to. We can be smart about it in the runtime and set the correct eventsourceid to the events committed in the `@compare` method.

As for where it belongs, I saw that this class also has all the other Runtime <-> SDK event conversions.

Reply:
I guess that it's fine for now. We'll keep it